org-code-javabuilder/lib/src/main/java/org/code/javabuilder/DashboardConstants.java

```diff
@@ -0,0 +1,10 @@
+package org.code.javabuilder;
+
+public final class DashboardConstants {
+  private DashboardConstants() {
+    throw new UnsupportedOperationException("Instantiation of constants class is not allowed.");
+  }
+
+  public static final String DASHBOARD_LOCALHOST_URL = "https://localhost-studio.code.org";
```

Comment:
I think my localhost uses `http` not `https` - can you do this in a way that supports either?

Reply:
Oh, actually, I'm realizing that we prepend `https` in `LambdaRequestHandler` - perhaps just removing that will solve things?

Reply:
Oh good catch, so does mine - I think this is here because we're manually adding an "https://" in LambdaRequestHandler, because I think previously it was a safe assumption that all incoming connections would be https, but not anymore. I'll just get rid of the protocol, since it's not relevant for this use case.